.github/workflows/next-build.yml

```diff
@@ -51,6 +51,20 @@ jobs:
         password: ${{ secrets.QUAY_PASSWORD }}
         registry: quay.io
 
+
+    - name: Release-test
+      run: |
+        # Need to grab goimports otherwise formatting after this step will fail
+        # PR checks.
+        go install golang.org/x/tools/cmd/goimports@latest
+
+        git config --global user.name "Andrew Obuchowicz"
+        git config --global user.email "aobuchow@redhat.com"
+
+        export GITHUB_TOKEN=test
+
+        ./make-release.sh --version 0.25.0
+
```

Comment:
Is this intentionally included in this PR?

Reply:
Oops no this is not intended to be apart of the PR - it's a result of me trying to test the release script on my fork just to make sure things work as expected before merging. This commit will be removed shortly.